### PR TITLE
Added back Add System button on system page

### PR DESCRIPTION
**Pull request by bisht-richa**
## What does this PR change?
Added back `Add System` button on System list page

**add description**

## GUI diff

No difference.

Before:

After:
Screenshot 2024-04-30 at 9 33 13

- [x] **DONE**

## Documentation
- No documentation needed

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Issue(s): [#24146](https://github.com/SUSE/spacewalk/issues/24146)
Port(s): # **add downstream PR(s), if any**

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: web/html/src/branding/css/susemanager/components/buttons.less [hunk 1]
```diff
@@ -35,6 +35,9 @@ button.is-plain,
   }
 }
 
+a[role="button"] {
```
Comment (Etheryte) [inline web/html/src/branding/css/susemanager/components/buttons.less:38]: We probably want the underline to be present or missing regardless of whether the element is a `button` tag or `a` tag, no? In other words, I would expect 

```tsx
<a href="..." role="button" className="btn btn-default">...</a>
```

and

```tsx
<button className="btn btn-default">...</button>
```

to look and behave the same, right now I'm guessing they don't?

Comment (bisht-richa) [reply web/html/src/branding/css/susemanager/components/buttons.less:38]: No, they don’t behave if I use a tag with role=‘button’, it shows an underline on the hover state. I am not sure if I answered your question correctly.

Comment (Etheryte) [reply web/html/src/branding/css/susemanager/components/buttons.less:38]: Yeah that was my comment exactly, we want those two to look the same, no?

Comment (bisht-richa) [reply web/html/src/branding/css/susemanager/components/buttons.less:38]: Yes

File: web/html/src/branding/css/susemanager/components/buttons.less [hunk 2]
```diff
@@ -35,7 +35,7 @@ button.is-plain,
   }
 }
 
-
+
```
Comment (Etheryte) [inline web/html/src/branding/css/susemanager/components/buttons.less:38]: Probably unintended?

File: web/html/src/manager/systems/all-list.tsx [hunk 3]
```diff
@@ -42,8 +42,13 @@ export function AllSystems(props: Props) {
         >
           <IconTag type="header-help" />
         </a>
-      </h1>
 
+        <div className="pull-right btn-group">
+          <a id="addsystem" title="Add a system" className="btn btn-link" href="/rhn/manager/systems/bootstrap">
+            <i className="fa fa-plus"></i>Add System
```
Comment (Etheryte) [inline web/html/src/manager/systems/all-list.tsx:48]: This looks good otherwise, but it needs a translation wrapper: `{t("Add System")}`

**Review by Etheryte — APPROVED** [review]

**Comment by Etheryte** [comment]
LGTM. 👍

**Review by Etheryte — APPROVED** [review]

**Review by Etheryte — APPROVED** [review]

**Pull request merged**